app/assets/stylesheets/ama_layout/layout_components/comparison-radios.scss

```diff
@@ -136,6 +136,32 @@
   &__divider{
     margin-bottom: 0;
   }
+
+  &--disabled{
+    input[type="radio"]:checked + label{
+      background-color: lighten($sky, 4%);
+      border: 2px solid lighten($dark-sky, 12%);
```

Comment:
Just wondering what the difference is between using lighten on the new variable, rather than just darkening the $sky variable? Does the checked state look different on these ones than the reg radio buttons?